lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -77,6 +77,7 @@
     privatekey_passphrase:
         description:
             - The passphrase for the I(privatekey_path).
+            - This is required if the privatekey is password protected.
```

Comment:
```suggestion
            - This is required if the private key is password protected.
```
Can you also adjust other occurences of `privatekey` in the documentation to `private key`? Or at least here and in the descriptions of `privatekey_path` and `privatekey_passphrase` in `openssl_csr`?

Reply:
👍